src/vue/components/EmbedPlaceholder/EmbedPlaceholder.vue

```diff
@@ -25,8 +25,8 @@
         </slot>
       </div>
       <slot name="button">
-        <consent-actions v-slot="{ consentCustomPurpose }">
-          <button class="embed-placeholder__button" @click.prevent="consentCustomPurpose(socialPurposeId)">
+        <consent-actions v-slot="{ customConsent }">
```

Comment:
Don't we need to use this customConsent method in the sharing popup as well?

Reply:
You'r right. I'm going to create another pull request that covers this component as well.